Sources/InstantSearch.swift

```diff
@@ -202,10 +202,50 @@ import UIKit
     }
     
     /// Get the searcher associated to the specific index with name and id used for InstantSearch
-    public func getSearcher(named name: String, withId id: String = "") -> Searcher? {
+    @objc public func getSearcher(named name: String, withId id: String = "") -> Searcher? {
         return searchers[SearcherId(index: name, variant: id)]
     }
     
+    // MARK: Caching
+    
+    /// Whether the search cache is enabled on this index. Default: `false`.
+    ///
+    @objc public var searchCacheEnabled: Bool = false {
```

Comment:
What if you only want for one index the cache to be enabled?

Reply:
Good point will fix